jerry-main/main-unix-snapshot.c

```diff
@@ -294,6 +308,11 @@ process_generate (cli_state_t *cli_state_p, /**< cli state */
     return JERRY_STANDALONE_EXIT_CODE_FAIL;
   }
 
+#ifdef JERRY_ENABLE_EXTERNAL_CONTEXT
+  jerry_context_t *context_p = jerry_create_context (512 * 1024, context_alloc, NULL);
```

Comment:
Why is the heap size hardcoded? Cannot we use `CONFIG_MEM_HEAP_AREA_SIZE` here?

Reply:
Good idea.